cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -8,10 +8,9 @@
                         "name": "Nexmo",
                         "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
                     },
-                    "created_on": "2000-01-01T00:00:00.000000000-00:00",
                     "text": "Ryan Lewis",
                     "urn": "tel:+12065551212",
-                    "uuid": "84f8a3cf-0f2c-4881-9502-2d7b114bf01f"
```

Comment:
if you change the number of calls to `utils.NewUUID`... all the UUIDs are going to change in the test data